.github/workflows/ci.yml

```diff
@@ -16,4 +16,4 @@ jobs:
         run: flake8 . || true # preventing CI failing while flake8 issues exist
 
       - name: flake8 validating correct file(s)
-        run: flake8 asv/main.py asv/repo.py asv/results.py asv/runner.py asv/statistics.py
\ No newline at end of file
+        run: flake8 asv/__init__.py asv/config.py asv/benchmarks.py
```

Comment:
Same question as in the other PR.

Reply:
This was to test the actual files I had added to this branch but I will remove it .